src/util/cmdlineargs.cpp

```diff
@@ -52,12 +53,23 @@ bool CmdlineArgs::Parse(int &argc, char **argv) {
         } else if (argv[i] == QString("--timelinePath") && i+1 < argc) {
             m_timelinePath = QString::fromLocal8Bit(argv[i+1]);
             i++;
+        } else if (argv[i] == QString("--debugLevel") && i+1 < argc) { // Set cmdline message verbosity
+            bool isInt; // Scoped variable, will be destroyed at the end of this block @vhexs
```

Comment:
Please remove this comment. There is nothing special about a simple local variable of type bool that has neither a constructor nor a destructor. It is just kicked off the stack when leaving the current scope. Comments should only be used for revealing something that is not directly visible from the code.

On the other hand I prefer to initialize such variables with a default value, even it should be set immediately by the following function call. You never know what might go wrong inside this function.